doc/reference/autoinstall-reference.rst

```diff
@@ -4,11 +4,35 @@ Autoinstall configuration reference manual
 ==========================================
 
 The autoinstall file uses the YAML format. At the top level, it must be a
```

Comment:
This part needs rephrased slightly, as while we strongly suggest using the top-level `autoinstall` and putting stuff under it, I plan to continue to support the existing format.  So the word 'must' here is not quite correct.

Reply:
Makes sense. I was trying to be a bit too forceful in moving users to the new format. I took out the `must` but left it unchanged as I think it still reads fine otherwise. Do you think this is sufficient with the note below that paragraph?